src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js

```diff
@@ -176,7 +177,7 @@ function SQFormGuidedWorkflow({
   });
 
   return (
-    <Section style={{padding: '20px'}}>
+    <Section style={containerStyles}>
```

Comment:
How do we feel about `style={{padding: '20px', ...containerStyles}}` which will always give the 20px padding without the consumer having to define it if they want to add _other_ styles. If the consumer provides `padding` it'll override the 20px. But if they provide, for instance, `margin` our current solution will then nuke the padding which would mean the consumer would need to provide that padding as well if they want to keep it.

Plus, then we could make the default value for `containerStyles` to be `{}` which reads a bit easier and could be a bit easier to maintain in the future.

Reply:
Agreed 💯 
I didn't notice the default padding was removed.